src/basic/log.c

```diff
@@ -1037,8 +1039,6 @@ int log_struct_iovec_internal(
                 size_t n_input_iovec) {
 
         PROTECT_ERRNO;
-        size_t i;
-        char *m;
```

Comment:
this is removed in the first commit, but the new lower level declaration is added in the second commit